.bazelrc

```diff
@@ -136,6 +136,7 @@ build:clang-asan-common --linkopt --rtlib=compiler-rt
 build:clang-asan-common --linkopt --unwindlib=libgcc
 
 build:clang-asan --config=clang-asan-common
+build:clang-asan --linkopt='-L/opt/llvm/lib/clang/14.0.0/lib/x86_64-unknown-linux-gnu'
```

Comment:
could you remove the entry from `rbe-toolchain-asan` please as that will no longer be necessary